workitem/workitem_repository.go

```diff
@@ -947,10 +947,19 @@ func (r *GormWorkItemRepository) getAllIterationWithCounts(ctx context.Context,
 	var allIterations []uuid.UUID
 	db.Pluck("id", &allIterations)
 	var res []WICountsPerIteration
-	db = r.db.Table(workitemTableName).Select(`iterations.id as IterationId, count(*) as Total,
-			count( case fields->>'system.state' when 'closed' then '1' else null end ) as Closed`).Joins(`left join iterations
```

Comment:
Agile space has `Closed` instead of `closed`. How was this even working?

Reply:
Who knows, maybe it wasn't. I just changed the query without worrying about this too much. We know that this code exists.

Reply:
Okay. But I suspect it isn't working as expected. I'll try adding a test in a separate PR.

Reply:
I can change the `=` to an `ILIKE` to make the comparison work in this case.

```sql
select count(*) FILTER (WHERE foo.a ILIKE foo.b) from (select 'closed'::text a, 'Closed'::text b) as foo;
```

Reply:
I've change it to `ILIKE` here cf8d8d5